### ISSUE TITLE
int8 communication quantization

### ISSUE THREAD

**Issue by HandH1998** [microsoft/Megatron-DeepSpeed#236]
Hello! Have you tried using int8 communication quantization in pipeline parallelism? As you may know, pipeline parallelism is often used across multiple machines, which typically have lower bandwidth than a single machine. Compressing the activations or gradients between machines can reduce the communication time and the total training time, but it may also impact the training loss and convergence. If you have tried this approach, could you please share how much it affects the training results? Do you think it is worth implementing this optimization? Thank you!
